app/controllers/usersController.js

```diff
@@ -5,7 +5,7 @@ const usersRepository = require( "../repositories/usersRepository" );
 
 const SECRET = "superSuperSecret";
 
-const register = async ( req, res ) => {
+const register = async ( req, res, next ) => {
```

Comment:
not related to your work but "async" is not needed on this functions since there is no await in it.